specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-12-01-preview/costmanagement.exports.json

```diff
@@ -0,0 +1,1136 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-12-01-preview",
```

Comment:
This file only contains a version update, but the apis remain the same, I don't think this api needs to be included.

Reply:
Removed it